packages-ts/gauntlet-terra-contracts/src/commands/abstract/executionWrapper.ts

```diff
@@ -14,12 +14,15 @@ export interface AbstractInstruction<Input, ContractInput> {
   validateInput: (input: Input) => boolean
   makeContractInput: (input: Input) => Promise<ContractInput>
   afterExecute?: (response: Result<TransactionResponse>) => any
+  isSimulateBeforeExecution?: boolean
```

Comment:
I think we will always want the simulation, can't think of a case where we don't want it